### PR TITLE
[zh] Fix typo in the number of types of hook handlers for Containers

### PR DESCRIPTION
**Pull request by jongwooo**
en PR #44499

Fix number of types of hook handlers available to implement for Containers.

### CODE REVIEW

**Comment by windsonsea** [comment]
Hi @jongwooo, this file has some more in-synced content with the en upstream, which you can find by running this command on your `main` branch:

```
./scripts/lsync.sh content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md
```
The check result is similar to:
```diff
+* Sleep - Pauses the container for a specified duration. 
+  The "Sleep" action is available when the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+  `PodLifecycleSleepAction` is enabled.
 
 ### Hook handler execution
 
 When a Container lifecycle management hook is called,
 the Kubernetes management system executes the handler according to the hook action,
-`httpGet` and `tcpSocket` are executed by the kubelet process, and `exec` is executed in the container.
+`httpGet` , `tcpSocket` and `sleep` are executed by the kubelet process, and `exec` is executed in the container.
```
Please include those differences into this PR too, thanks.

**Comment by jongwooo** [comment]
> Please include those differences into this PR too, thanks.

@windsonsea Sure. I may make some grammatical errors as I am not fluent in Chinese😅

File: content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -86,34 +86,40 @@ A more detailed description of the termination behavior can be found in
...
 * HTTP - Executes an HTTP request against a specific endpoint on the Container.
+* Sleep - Pauses the container for a specified duration.
+  The "Sleep" action is available when the [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+  `PodLifecycleSleepAction` is enabled.
 -->
 
 * Exec - 在容器的 cgroups 和名字空间中执行特定的命令（例如 `pre-stop.sh`）。
   命令所消耗的资源计入容器的资源消耗。
 * HTTP - 对容器上的特定端点执行 HTTP 请求。
+* Sleep - 在指定时间内暂停容器。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md:108]:
```suggestion
* Sleep - 将容器暂停一段指定的时间。
```

File: content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md [hunk 2]
```diff
@@ -86,34 +86,40 @@ A more detailed description of the termination behavior can be found in
...
+  当 [特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/) `PodLifecycleSleepAction` 被启用时，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md:109]:
```suggestion
  当[特性门控](/zh-cn/docs/reference/command-line-tools-reference/feature-gates/) `PodLifecycleSleepAction` 被启用时，
```

File: content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md [hunk 3]
```diff
@@ -86,34 +86,40 @@ A more detailed description of the termination behavior can be found in
...
+  “Sleep” 动作可用。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md:110]:
```suggestion
  “Sleep” 操作才可用。
```

**Comment by jongwooo** [comment]
@windsonsea Thank you for your kind review:)

**Review by windsonsea — COMMENTED** [review]
/approve

**Pull request merged**